app/frontend/src/features/auth/signup/CompleteSignupForm.tsx

```diff
@@ -319,6 +320,11 @@ export default function CompleteSignupForm() {
                 {acceptedTOS ? THANKS : ACCEPT}
               </Button>
             </div>
+            {isLocationEmpty && (
```

Comment:
Can you add better error handling while you're here?
Change `isLocationEmpty` to just `error`, reset it on submit, and catch any errors from `authActions.signup()`.